apps/prairielearn/src/tests/sync/util.ts

```diff
@@ -470,12 +472,15 @@ export async function createAndSyncCourseData() {
  * path to the directory.
  *
  * @param courseData - The course data to write and sync
- * @returns The path to the new temp directory
+ * @returns The path to the new temp directory and the sync results
  */
-export async function writeAndSyncCourseData(courseData: CourseData): Promise<string> {
+export async function writeAndSyncCourseData(courseData: CourseData): Promise<{
+  courseDir: string;
+  syncResults: syncFromDisk.SyncResults;
+}> {
   const courseDir = await writeCourseToTempDirectory(courseData);
-  await syncCourseData(courseDir);
-  return courseDir;
+  const syncResults = await syncCourseData(courseDir);
+  return { courseDir, syncResults };
```

Comment:
This is nice, I don't know why I didn't think to do this when I was writing these tests in the first place.

Reply:
I think this is the first time we're using these utils outside of the syncing test, so I don't blame you for not thinking of it. I didn't either!